bundle/Command/ProfilerCachePurgerCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ProfilerCachePurgerCommand extends ContainerAwareCommand
```

Comment:
It would be better to use DI to inject the dependencies (thereby creating the service for the command) instead of using `ContainerAwareCommand`.